packages/common/index.ts

```diff
@@ -634,6 +635,30 @@ export class AsyncQuestionVotes {
   vote!: number
 }
 
+export class AsyncQuestionComment {
+  @IsOptional()
+  @IsInt()
+  id?: number
+
+  @IsInt()
+  questionId!: number
+
+  @IsInt()
+  userId!: number
+
+  @Type(() => UserPartial)
+  creator!: UserPartial
+
+  @IsString()
+  commentText!: string
+
+  @IsString()
+  createdBy!: string
+
+  @IsDate()
+  createdAt!: number
+}
+
```

Comment:
As a tip for the future with types, have one type for creating comments (like AsyncQuestionCommentParams) which has all the decorators (which will get auto-validated with requests) as well as one type for the returned values (could just do AsyncQuestionComment) that has no decorators (as we don't need to validate the values coming from our own endpoint)